### ISSUE TITLE
Support new Maker Vault types

### ISSUE THREAD

**Issue by kilrau** [rotki/rotki#3952]
## Abstract

```
Detected vault with collateral_type WBTC-C. That is not yet supported by rotki. Skipping...
```

## Motivation

Would want this vault type to be supported by rotki, older WBTC types like e.g. WBTC-A are supported. Skimming through https://vote.makerdao.com/executive, it looks like this was the only new vault type since August 2021, more at https://vote.makerdao.com/executive/template-executive-vote-onboarding-new-vault-types-parameter-changes-and-core-unit-budget-distributions-november-26-2021?network=mainnet#proposal-detail

## Specification

Should be clear I think

**Comment by LefterisJP** [comment]
Yeah no worries @kilrau. Adding it for next release. It's so minor, that may even add it to a patch release.

**Comment by LefterisJP** [comment]
Should be fixed by #3958
